tests/models/test_ftrl.py

```diff
@@ -287,7 +299,17 @@ def test_ftrl_set_individual():
     ft.negative_class = tparams.negative_class
     ft.interactions = tparams.interactions
     ft.model_type = tparams.model_type
-    assert ft.params == tparams
+
+
+def test_ftrl_set_individual_after_params():
+    ft = Ftrl()
+    params = ft.params
+    ft.alpha = tparams.alpha
+    assert params == Ftrl().params
+    assert (ft.alpha, params.beta, params.lambda1, params.lambda2, params.nbins,
+           params.mantissa_nbits, params.nepochs, params.double_precision,
+           params.negative_class, params.interactions,
+           params.model_type) == ft.params
```

Comment:
I think you missing a check that `ft.params` still remains a namedtuple

Reply:
I'm doing that in `test_ftrl_create_params` and `test_ftrl_get_parameters`, but surely we can also do it here.

Reply:
`test_ftrl_get_params` checks for a simple tuple, not a named tuple. And so does `test_ftrl_create_params`. This is because a namedtuple compares equals to a regular tuple if their components match.

Here I was merely suggesting to add an extra check that after assignment, the `.params` remain a named tuple (this could have failed if we'd forgotten to re-assign tuple's type in `copy_v()`)

Reply:
Yes, I noticed that at some point but looks like never addressed in `test_ftrl_get_params` and `test_ftrl_create_params`. This check should also be there. It looks like the easiest way to do this kind of check is just to add component-wise asserts.